pyk/src/pyk/kcfg/exploration.py

```diff
@@ -109,4 +111,4 @@ def to_dict(self) -> dict[str, Any]:
 
     # Minimizing the KCFG
     def minimize_kcfg(self) -> None:
-        self.kcfg.minimize()
+        KCFGMinimizer(self.kcfg).minimize()
```

Comment:
proof.minimize_kcfg() with the same effect.

Reply:
This change doesn't seem necessary, as `KCFG` still provides the `minimize` method.

Reply:
Thank you for pointing this out. I'll remove this change.